apps/src/javalab/JavalabEditor.jsx

```diff
@@ -659,11 +662,57 @@ class JavalabEditor extends React.Component {
   }
 }
 
+const styles = {
+  editor: {
+    width: '100%',
+    height: '400px',
```

Comment:
Rather than setting the height here, you can use defaultProps. Example: https://github.com/code-dot-org/code-dot-org/blob/8d4a1e235c8867ff6a831d2db77c0713eab4b505/apps/src/templates/SchoolNotFound.jsx#L75
Docs: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Also, IIRC, you should be able to use height without the "px" as react will automatically add that to the style as a default unit.

Reply:
Updated!